sopel/config/core_section.py

```diff
@@ -1213,6 +1213,66 @@ def homedir(self):
 
     """
 
+    tls_ciphers = ValidatedAttribute(
```

Comment:
You can replace that by a `ListAttribute`, and then use it with a join like:

    `ssl_ctx.set_ciphers(':'.join(self.tls_ciphers))

If someone provide only one line with `:`, it will still work, so there is only advantages here.

Reply:
Good call